pkgs/tools/security/quark-engine/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, gitMinimal
+, python3
+}:
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+
+      # Requires the latest androguard release
+      androguard = super.androguard.overridePythonAttrs (oldAttrs: rec {
+        version = "3.4.0a1";
```

Comment:
Can we update the one in nixpkgs instead?

Reply:
We can.